### ISSUE TITLE
Add Dark Mode 

### ISSUE THREAD

**Issue by Gnesmith21** [jimradford/superputty#885]
Can we please add dark mode to super putty?

**Comment by waukeerules** [comment]
Yes please! Only downside is if you have custom icons that are also dark - you'll have to adjust them.
